git_machete/tests/functional/test_machete.py

```diff
@@ -2943,7 +2828,5 @@ def test_clean(self) -> None:
         )
         self.assert_command(['status'], expected_status_output)
 
-        with self.assertRaises(
-                subprocess.CalledProcessError,
-                msg="Verify that 'git checkout mars' raises an error when branch mars is no longer present in git."):
+        with pytest.raises(subprocess.CalledProcessError):
```

Comment:
Btw! I think this PR is already self-contained... so it can be merged without any adding extra new refactor (they can go onto subsequent PRs), in the spirit of git-machete ;) https://slides.com/plipski/git-machete/#/2 and following slides

Reply:
Yes, indeed! There is no sense and no benefit to making a PR of hundreds of lines of changes. I will implement it step by step.